modules/beta-private-cluster/variables.tf

```diff
@@ -968,3 +968,9 @@ variable "fleet_project_grant_service_agent" {
   type        = bool
   default     = false
 }
+
+variable "monitoring_metric_writer_role" {
```

Comment:
Please see https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/main/CONTRIBUTING.md#templating as these changes will also need to be made in the original template.

Reply:
Done